packages/block-editor/src/components/inserter/index.js

```diff
@@ -30,18 +44,34 @@ const defaultRenderToggle = ( {
 	hasSingleBlockType,
 	toggleProps = {},
 	prioritizePatterns,
+	label,
+	rootClientId,
 } ) => {
-	let label;
-	if ( hasSingleBlockType ) {
-		label = sprintf(
+	const {
+		getBlock,
+	} = useSelect( blockEditorStore );
```

Comment:
`defaultRenderToggle` isn't a component, so I'm not sure if hooks are ok. It might break the rule of hooks if the render function is conditionally called.

Reply:
Yeh I used the hook in the revised PR 👍